src/ILCompiler.MetadataWriter/src/Internal/Metadata/NativeFormat/Writer/LowLevelLinq.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Then I would need to invent a neutrally sounding namespace for it, which I wanted to avoid. If you have suggestions, I'll gladly move it.